### PR TITLE
watermarks and comments

### PR DESCRIPTION
**Pull request by jadedtdt**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -8,24 +8,13 @@
...
-	text-indent: 400x;
-}
-
-h1 {
-	text-indent: 100x;
-	text-align: justify;
-	margin-right: auto;
-	margin-left: 300x;
-}
-
```
Comment (emanguy) [inline index.php:21]: Did you mean to remove this CSS? Why did you do this?

File: index.php [hunk 2]
```diff
@@ -44,23 +33,28 @@
...
 </center>
+
+<!-- places a box around the login elements -->
 <div class = "loginBox">
 <h1>
+	<!-- input fields fName, lName, StudentID -->
 	<font size="5" face="Serif">
 	<br>
-	FIRST NAME: <input type="text" size="12" maxlength="20" name="tfNameF" required><br>
+	FIRST NAME: <input type="text" size="12" maxlength="20" name="tfNameF" placeholder="John" required><br>
```
Comment (emanguy) [inline index.php]: I would modify the placeholder here, make it obvious that this is an example.

``` html
<input type="text" size="12" maxlength="20" name="tfNameF" placeholder="example: John" required><br>
```

Same goes for the other two text fields.

File: index.php (hunk 1 above)
Comment (jadedtdt) [inline index.php:21]: was unnecessary

**Comment by emanguy** [comment]
Looks good. Merging now. Also I get notifications when you add commits to a pull request so don't worry about tagging me in a new comment :smile:

**Pull request merged**